tools/hybrid-quickstart/cloudbuild/hybrid.cloudbuild.yaml

```diff
@@ -0,0 +1,104 @@
+#  Copyright 2021 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# While the quickstart creates all required resources from scratch,
+# this script makes the following assumptions to reduce the permissions
+# on the cloud build SA. To prepare your project you can use the
+# corresponding steps in the steps.sh:
+# 1. enable_all_apis - All required Google APIs are enabled on the project
+# 2. create_sa - All Apigee service accounts created
+#
+# Cloud build service account needs the following roles:
+# 1. roles/editor
+# 2. roles/container.clusterAdmin
+# 3. roles/container.admin
+# 4. roles/iam.serviceAccountKeyAdmin
+steps:
+# Step 1: Install node dependencies for linting and testing
+- name: gcr.io/cloud-builders/gcloud
+  entrypoint: "bash"
+  args:
+    - "-c"
+    - |-
+      # git clone https://github.com/apigee/devrel.git
+
+      git clone https://github.com/danistrebel/devrel.git && cd devrel
```

Comment:
feature branch

Reply:
Gonna remove this and expect an invocation with the source context instead. This makes it more flexible.